src/identity/components/GlobalHeader/GlobalHeader.tsx

```diff
@@ -77,42 +77,44 @@ export const GlobalHeader: FC = () => {
     }
   }, [orgsList])
 
-  const shouldLoadDropdowns = activeOrg?.id && activeAccount?.id
-  const shouldLoadAvatar =
-    user?.firstName && user?.lastName && user?.email && org?.id
-  const shouldLoadGlobalHeader = shouldLoadDropdowns || shouldLoadAvatar
+  const shouldLoadDropdowns = !!(activeOrg?.id && activeAccount?.id)
+  const shouldLoadAvatar = !!(
```

Comment:
if we're already wrapping this boolean cast in parens, might as well do something like

```ts
Boolean(activeOrg?.id && activeAccount?.id)
```

which does the same thing as

```ts
!!(activeOrg?.id && activeAccount?.id)
```

but is generally more explicit and clear to people who aren't super familiar with JavaScript

Reply:
No, I like your change - there's a mix of both `!!` and `Boolean` in the codebase, but `Boolean` is definitely clearer. Will change.